Lora1

LoraShk1 opened the pull request:
Added animal and number game

Dani-gr commented:
![](https://github.trello.services/images/mini-trello-icon.png) [Practice mode, two games (for now, copying the other game, maybe creating another)](https://trello.com/c/qYvWry5m/11-practice-mode-two-games-for-now-copying-the-other-game-maybe-creating-another)

Dani-gr commented on ClientApp/practiceactivity2.html:
We should change the alt text of Umberto.

LoraShk1 replied:
The umberto img is just a placeholder honestly because it gets exchaged with another image immediatly... So i dont know if its that important

Dani-gr replied:
Oh, okey then

Dani-gr commented on ClientApp/practiceactivity3.html:
Same here, we should change the alt text of Umberto.

Dani-gr commented on ClientApp/js/colorGame.js:
Los cambios de este archivo son accidentales, no deberían realizarse.

Dani-gr reviewed (changes requested):
It works well, but the menu is messed up (maybe related to commonStyles?), and my game doesn't work (just reverting those changes should be fine).

xmariafdz commented:
main menu's buttons are broken :(

LoraShk1 commented:
> It works well, but the menu is messed up (maybe related to commonStyles?), and my game doesn't work (just reverting those changes should be fine).

I think i fixed the issue with your game in the last commit

LoraShk1 commented:
> main menu's buttons are broken :(

you mean not linking to the right target or design is broken? because the design is not fully finished i think

LoraShk1 commented:
> main menu's buttons are broken :(

you mean not linking to the right target or design is broken? because the design is not fully finished i think

Dani-gr commented:
> > It works well, but the menu is messed up (maybe related to commonStyles?), and my game doesn't work (just reverting those changes should be fine).
> 
> I think i fixed the issue with your game in the last commit

Hmm, I still see it the same way, did you push the commit?

LoraShk1 commented:
> > > It works well, but the menu is messed up (maybe related to commonStyles?), and my game doesn't work (just reverting those changes should be fine).
> > 
> > 
> > I think i fixed the issue with your game in the last commit
> 
> Hmm, I still see it the same way, did you push the commit?

what exactly is not working because for me the games work

Dani-gr commented:
> > > > It works well, but the menu is messed up (maybe related to commonStyles?), and my game doesn't work (just reverting those changes should be fine).
> > > 
> > > 
> > > I think i fixed the issue with your game in the last commit
> > 
> > 
> > Hmm, I still see it the same way, did you push the commit?
> 
> what exactly is not working because for me the games work

The color game doesn't work (there aren't any spaces), since all the uses of the variable `color` have been changed to `number`. If it works for you, check if the commit hasn't been pushed yet

LoraShk1 commented:
> > > > > It works well, but the menu is messed up (maybe related to commonStyles?), and my game doesn't work (just reverting those changes should be fine).
> > > > 
> > > > 
> > > > I think i fixed the issue with your game in the last commit
> > > 
> > > 
> > > Hmm, I still see it the same way, did you push the commit?
> > 
> > 
> > what exactly is not working because for me the games work
> 
> The color game doesn't work (there aren't any spaces), since all the uses of the variable `color` have been changed to `number`. If it works for you, check if the commit hasn't been pushed yet

i did - thats exactly what i fixed. I commited again maybe its better now ?

Dani-gr commented:
It still doesn't work for me, I'll try to revert the changes on that file
![image](https://user-images.githubusercontent.com/100536136/231898331-a3db9ef4-2f29-4604-9fe1-531ddfd7da6f.png)

Dani-gr reviewed (approved):
Now it works for me, lmk if it does for you too

xmariafdz commented:
both games work for me! Have the same issue with the menu, but we can fix it later

Dani-gr commented:
Since it seems ok, I'll proceed with the merge.
